components/extractors/storage/index.md

```diff
@@ -9,6 +9,7 @@ Extractors import data from external sources and integrate it to the Keboola Con
 The following extractors allow access to data from generic storage services:
 
 - [AWS S3](/components/extractors/storage/aws-s3) --- imports CSV files from multiple AWS S3 buckets into multiple tables with additional postprocessing.
+- [Azure Datalake Gen2](/components/extractors/storage/azure-datalake-gen2) --- imports CSV files from Azure Datalake Gen2 into multiple tables with additional postprocessing.

```

Comment:
I suggest removing ` with additional postprocessing.` , it's unnecessarily long.